docs/eigenda/operator-guides/run-a-node/registration.md

```diff
@@ -0,0 +1,76 @@
+---
+sidebar_position: 2
+---
+
+# Register Your Operator
+
+Your operator will not begin receiving traffic from the EigenDA disperser until it has registered for one or more quorums with EigenDA. Note, as discussed in [delegation requirements](../requirements/delegation-requirements/), that registration with an EigenDA quorum requires that an operator already be [registered as an operator with EigenLayer](https://docs.eigenlayer.xyz/operator-guides/operator-installation) and to have a minimum amount of stake delegated within each quorum to be registered. 
+
+## Opt-in to an EigenDA Quorum
+
+If you meet the delegation requirements for opting into one or more quorums, you can execute the following command from the `eigenda-operator-setup/mainnet` folder to opt-in to the desired quorums:
```

Comment:
Is this meant to ask the user to review the eigenda-operator-setup/mainnet folder for further instructions?

Could we clarify the meaning behind each integer value here? E.g. quorum 0 represents LSTs, quorum 1 represents WETH, etc.? Or should those mappings be managed elsewhere?

Reply:
I've added a tab group which shows only the information relevant to the network.

Reply:
looks good